configs/hours.yaml

```diff
@@ -112,5 +112,5 @@ holidays:
     reason: Public Health Closure
   - date: [2020-03-21, 2020-03-29]
     reason: Spring Break
-  - date: [2020-03-30, 2020-05-03]
+  - date: [2020-03-30, 2020-05-31]
```

Comment:
Perhaps it might be worth splitting this into "Public health closure" until Spring ends, then add a new break for Summer.

Reply:
no review, only merge